data_steward/cdr_cleaner/cleaning_rules/clean_mapping.py

```diff
@@ -181,6 +181,12 @@ def setup_rule(self):
         """
         pass
 
+    def get_sandbox_tablenames(self):
+        """
+        Returns a list of sandbox table names.
+        """
+        raise NotImplementedError("Please fix me")
```

Comment:
I did mean to leave this in.  The abstractmethod `get_sandbox_tablenames` was added to the base class after this ticket adding this cleaning rule was completed.  Because this ticket didn't implement this, the unit tests for this class failed.  I didn't write that code and am not the best person to fill in this list of sandbox table names.  So, I added this in to allow the unit tests to pass and to alert the next developer to implement this function.  Raising the error is a better alert than returning an empty list.

Reply:
<img width="563" alt="Screen Shot 2020-04-10 at 2 23 22 PM" src="https://user-images.githubusercontent.com/47253368/79017449-f9588580-7b36-11ea-9238-509351e22937.png">
Adding this comment.  Good?